leetcodePython/TwoPointers/twopointers_75.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+
+class SolutionCountSort:
+    def sortColors(self, nums: List[int]) -> None:
+        count_0, count_1, count_2 = 0, 0, 0
+        for num in nums:
+            if num == 0:
+                count_0 += 1
+            elif num == 1:
+                count_1 += 1
+            else:
+                count_2 += 1
+        for i in range(len(nums)):
+            if count_0 > 0:
+                nums[i] = 0
+                count_0 -= 1
+            elif count_1 > 0:
+                nums[i] = 1
+                count_1 -= 1
+            else:
+                nums[i] = 2
+
+
+class SolutionDutchNationalFlag:
+    def sortColors(self, nums: List[int]) -> None:
+        i1, i2, i3 = 0, 0, len(nums) - 1
+        num1, num2, num3 = 0, 1, 2
```

Comment:
## [Dutch national flag problem](https://en.wikipedia.org/wiki/Dutch_national_flag_problem)

Idea
* i1 be the right end of number1 sequence
* i3  be the left end of number3 sequence
* i2 be the current index